lib/evm_database.rb

```diff
@@ -77,8 +77,26 @@ def self.seed_rest
     seed(OTHER_SEEDABLE_CLASSES)
   end
 
+  # Returns whether or not a primordial seed has completed.
+  def self.seeded_primordially?
+    # While not technically accurate, as someone could just insert a record
+    # directly, this is the simplest check at the moment to guess whether or not
+    # a primordial seed has completed.
+    MiqDatabase.any?
```

Comment:
Seeing the `seeded?` method "care" about regions makes me want to use a table that isn't excluded from replication here.  But, `MiqRegion.my_region` calls `MiqDatabase.first` anyway so I'll just leave this note here.  I think this is fine as is

Reply:
I can add MiqRegion as well to the list.  No reason we can't have 2 classes.